contextualhelp/data.json

```diff
@@ -1,2993 +1,3206 @@
+          }
+        ]
+      },
+      {
+        "route_name": "PAYMENT_ONBOARDING_SELECT_METHOD",
+        "title": "Qui puoi aggiungere un nuovo metodo di pagamento che verrà salvato nella sezione Portafoglio. Per rimuovere un metodo già inserito, vai alla sezione Portafoglio e seleziona il metodo desiderato.",
+        "content": "Qui puoi aggiungere e gestire i tuoi metodi di pagamento e i bonus disponibili su IO",
+        "faqs": [
+          {
+            "title": "Quali metodi di pagamento posso aggiungere al Portafoglio?",
+            "body": "Puoi aggiungere e salvare questi metodi di pagamento:\n- Carte di credito o di debito (anche prepagate) del circuito American Express, Maestro, Mastercard, Visa e Visa Electron.\n- PayPal, con pagamento veloce.\n\nSe vuoi usare un altro metodo, puoi sceglierlo al momento del pagamento, ma non verrà salvato nel Portafoglio per i pagamenti futuri."
+          },
+          {
+            "title": "Posso rimuovere un metodo di pagamento?",
+            "body": "Sì. I tuoi metodi di pagamento sono visualizzati come card nella parte alta dello schermo nella sezione Portafoglio. Seleziona la card del metodo che vuoi eliminare, poi premi su 'Elimina questo metodo'."
+          },
+          {
+            "title": "Perché il salvataggio della carta non va a buon fine?",
+            "body": "Può dipendere da diversi motivi, ma in ogni caso ti invitiamo a verificare con la tua banca.\nI casi più frequenti sono:\n1. La tua carta non è abilitata agli acquisti online.\n2. Non hai ancora attivato il servizio 3DS: si tratta di un sistema di sicurezza legato ai pagamenti online.\n3. La tua carta è stata sospesa o bloccata.\n4. Hai messo ‘in pausa’ la tua carta."
+          },
+          {
+            "title": "Posso pagare anche a rate con PayPal?",
+            "body": "Sì, ma non con il metodo salvato nel Portafoglio.\n\nPer pagare a rate, seleziona il metodo ‘PayPal (anche in 3 rate)’ durante il pagamento e segui le indicazioni.\n\nQuesta opzione è disponibile, dal 25 luglio, solo per alcuni avvisi. Per verificare se il tuo avviso è idoneo, consulta i [Termini e condizioni d’uso di Paga in 3 Rate - PayPal](https://www.paypal.com/it/webapps/mpp/campaigns/pay-in-3-full)."
+          },
+          {
+            "route_name": "PAYMENT_TRANSACTION_BIZ_EVENTS_LIST_SCREEN",
+            "title": "Ricevute pagoPA",
+            "content": "Qui puoi vedere le ricevute dei pagamenti pagoPA e, se disponibile, salvare o condividere la ricevuta in formato PDF.",
+            "faqs": [
```

Comment:
I was just passing by and noticed that a route has been placed inside a faq array, possibly by mistake. This will break the contextual data JSON decoder, causing the checks to fail. Here is the correct diff, I guess. Please review it, preferably by ignoring whitespaces for a clearer comparison.
[contextualHelpData.patch](https://github.com/user-attachments/files/16317266/contextualHelpData.patch)